applications/crossbar/src/crossbar_maintenance.erl

```diff
@@ -432,7 +432,6 @@ validate_account(JObj, Context) ->
                                  ,[{fun cb_context:set_req_data/2, JObj}
                                   ,{fun cb_context:set_req_nouns/2, [{?KZ_ACCOUNTS_DB, []}]}
                                   ,{fun cb_context:set_req_verb/2, ?HTTP_PUT}
-                                  ,{fun cb_context:set_resp_status/2, success}
```

Comment:
why the remove of this ? it will use the current resp_status which may be `success`